mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,17 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.deleteOrderById = (req, res) => {
```

Comment:
The function name `deleteOrderById` is not following the conventional naming convention of JavaScript, which is camelCase. It should be renamed to `deleteOrderId`.